examples/pbam/toy_electrostatic.in

```diff
@@ -3,17 +3,20 @@ read
     mol pqr neg_charge.pqr
 end
 elec name comp_solv        # Toy charges
-    pbam
+    pbam-auto
 
     runtype electrostatics                            # Can be energyforce, electrostatics etc
     runname elec_toy                                  # Output name for files created by program
     temp 298.15                                       # System temperature (K)
     pdie 4.0                                          # Solute dielectric                 
     sdie 78.0                                         # Solvent dielectric                
-    ion charge 0.0 conc 0.05 radius 0.0               # Monovalent salt conc in M
+    # WARN: ion is not supported in pbam-auto
+    # ion charge 0.0 conc 0.05 radius 0.0               # Monovalent salt conc in M
```

Comment:
@sobolevnrm should `ion` be part of the pbam-auto specification? It isn't in the current documentation.

Reply:
I don't think so but I don't know pbsam very well. @lfelberg is the person to ask if she has time.

Reply:
I will leave the WARN in the input file and add it to the list of keywords to deprecate. Hopefully we will hear back from @lfelberg and get guidance on how to fix the example.